### PR TITLE
Drop show-branch

### PR DESCRIPTION
**Pull request by bokuweb**
- Remove `git show-branch` command.
- Improve performance.

Relevant issue: #84 

I guessed this issue caused by performance.
Therefore, I would like to see how it goes by this fixes.

### CODE REVIEW

**Comment by Quramy** [comment]
@bokuweb I think you've some mistakes. See [Circle CI log of your commit's](https://circleci.com/gh/reg-viz/reg-suit/326?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):

```text
[reg-suit] info Detected the previous snapshot key: 'ace809e689db2046e17ac348aae985bce062a997'
``` 

The ace809e689db2046e17ac348aae985bce062a997 hash points "the current" itself...

**Comment by bokuweb** [comment]
@Quramy  Could you please review this PR, if you get a chance.

File: packages/reg-keygen-git-hash-plugin/src/git-cmd-client.ts [hunk 1]
```diff
@@ -23,23 +23,19 @@ export class GitCmdClient {
     return execSync(`git branch -a --contains ${hash}`, { encoding: "utf8" });
   }
 
-  showBranch() {
-    return execSync("git show-branch -a --sha1-name", { encoding: "utf8" });
```
Comment (Quramy) [inline packages/reg-keygen-git-hash-plugin/src/git-cmd-client.ts]: nice!

**Comment by Quramy** [comment]
Thx! 😄

**Pull request merged**